src/app/(privateProfile)/savings/page.tsx

```diff
@@ -1,8 +1,274 @@
+							</div>
+						)}
+					</div>
+				</li>
+			);
+		});
+	}
+
+	const renderSavingsTransactions = (transactions: ISavingsTransaction[]) => {
+		return transactions.map((savingsData, index) => (
+			<li key={index}>
+				<SavingsTransaction
+					firstDate={savingsData.firstDate}
+					secondDate={savingsData.secondDate}
+					purpose={savingsData.purpose}
+					sum={savingsData.sum}
+				/>
+			</li>
+		));
+	};
+
 	return (
-		<>
-			<div>Savings</div>
-		</>
+		<div className={style.savingsPageWrap}>
+			<div className={style.savingsPageContainer}>
+				<form className={style.savingsFormContentWrapper}>
+					<h1 className={style.headerTitle}>Накопления</h1>
+					<div className={style.savingsFormContentWrapperChoice}>
+						<div className={style.savingsByDateContainer}>
+							<div className={style.totalAmountWrapper}>
+								<p className={style.totalAmountWrapper__savings}>Общая сумма накоплений </p>
+								<p className={style.totalAmountWrapper__sum}>4 112 500 ₽</p>
+							</div>
+							<div className={style.dateSelectionWrapper}>
+								<p className={style.dateSelectionWrapper__description}>Выбор даты</p>
+								<AppInput control={control} label={"Выбор даты"} type={InputTypeList.Date} name={"date"} />
+							</div>
+						</div>
+						<div className={style.savingsDetailsContainer}>
+							<div className={style.savingsDetailsContainer__category}>
+								<Select
+									name={"expenses"}
+									label={"Накопления"}
+									options={["Обучение ребенка", "Машина", "Квартира", "Отпуск 2024"]}
+								/>
+							</div>
+							<div className={style.savingsDetailsContainer__sum}>
+								<AppInput
+									control={control}
+									label={"Сумма"}
+									type={InputTypeList.Number}
+									name={"number"}
+									placeholder={"0.00 ₽"}
+								/>
+							</div>
+
+							<Button content={"Добавить"} styleName={"buttonForSavings__disabled"}>
+								<PlusIcon classNames={style.addButtonIcon} />
+							</Button>
+						</div>
+					</div>
+					<div className={style.savingsFormContentWrapperList}>
+						<div className={style.wrapperList__header}>
+							<ul className={style.wrapperListHeaderBlock}>
+								<li className={style.wrapperListHeaderBlock__category}>Категория</li>
+								<li className={style.wrapperListHeaderBlock__target}>
+									<div className={style.wrapperListHeaderBlock__targetPosition}>
+										<p>Цель, ₽</p>
+										<div onClick={handleSortByTarget} role="button">
+											<SortIcon classNames={style.sortIcon} />
+										</div>
+									</div>
+								</li>
+								<li className={style.wrapperListHeaderBlock__sum}>
+									<div className={style.wrapperListHeaderBlock__sumPosition}>
+										<p>Сумма, ₽</p>
+										<div className={style.sortIcon} onClick={handleSortBySum} role="button">
+											<SortIcon />
+										</div>
+									</div>
+								</li>
+								<li className={style.wrapperListHeaderBlock__status}>Статус</li>
+							</ul>
```

Comment:
can you add horizontal scroll for wrapperListHeaderBlock block resolving error mobile view?